src/information_theory/hamming.lean

```diff
@@ -37,7 +37,10 @@ def hamming_dist (x y : Π i, β i) : ℕ := (univ.filter (λ i, x i ≠ y i)).c
 
 /-- Corresponds to `dist_self`. -/
 @[simp] lemma hamming_dist_self (x : Π i, β i) : hamming_dist x x = 0 :=
-by { rw [hamming_dist, card_eq_zero, filter_eq_empty_iff], exact λ _ _ H, H rfl }
+begin
+  rw [hamming_dist, card_eq_zero, filter_eq_empty_iff],
+  exact λ _ _ H, H rfl
+end
```

Comment:
Not sure this is an improvement. When it fits on a line, it fits on a line.

Reply:
Ping

Reply:
Oh, huh, I thought I dealt with this. I'm away on holiday currently so it's difficult to make this kind of change.